modules/libcom/src/as/asLib.h

```diff
@@ -173,11 +180,27 @@ typedef struct{
     ELLNODE	node;
     char	host[1];
 } HAGNAME;
+
 typedef struct hag{
 	ELLNODE	node;
 	char	*name;
 	ELLLIST	list;	/*list of HAGNAME*/
 } HAG;
+
+/* A sorted list of unique integers used to hold 
+ * IP addresses */
+typedef struct addrlist{
+        epicsUInt32 *addrs;
```

Comment:
Why does "list of list of IP addresses" need to be separate from "list of host names"?